text/0032-nft-standard.md

```diff
@@ -146,8 +251,16 @@ There are some special cases where the NFT token won't follow the proposed stand
 
 Given that this situation is expected to be rare, we will handle them manually. The NFT creator will need to get in touch with Hathor team, in order to have the token identified as an NFT on the official Hathor explorer. Besides that, as long as the digital asset's URL is immutable, it should be approved in the review and should be shown in Hathor's Public Explorer like any other standard NFT.
 
+## Drawbacks
```

Comment:
I would remove this section entirely. Drawbacks for this version should be in the comments of this PR, not the final RFC.